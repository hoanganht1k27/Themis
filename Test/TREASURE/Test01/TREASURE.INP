16 10
49 15 37 29 51 99 87 85 64 64 37 61 81 91 30 90
12 10
1 11
8 6
7 13
1 2
13 12
8 14
11 5
16 1
2 7
2 4 8 7 5 6 1 3 10 9
