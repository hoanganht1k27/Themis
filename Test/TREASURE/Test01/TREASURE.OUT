673
612
561
470
421
340
276
177
90
0
